common/browser_context.go

```diff
@@ -27,6 +27,42 @@ import (
 // with the browserContext.waitForEvent API.
 type waitForEventType string
 
+// Cookie represents a browser cookie.
+//
+// https://datatracker.ietf.org/doc/html/rfc6265.
+type Cookie struct {
```

Comment:
I wonder if types such as `Cookie` and `ConsoleMessage` should be in their own files for better organization. At the same time, considering the "small associated implementation" and that they are pretty much tight to `BrowserContext`/`Page`, I can see them placed along with these types (as they are), so no strong opinion.

Reply:
Keeping them close to where they are used makes more sense to me. I'd like to keep them as they are. WDYT?